ops/README.md

```diff
@@ -18,6 +18,52 @@ Blockchain <-> Relay <-> CL node <-> price adapters
 - [Solana Test Validator](#local-solana-testnet)
 
 ## Usage
+Set a `.env` file inside of `chainlink-solana/gauntlet`. The private key can be generated the same way as the program accounts (`solana-keygen`) then copied into the env file.
+```
+# .env
+PRIVATE_KEY=[96,13,...]
+SECRET=some random local testing secret
+```
+
+Generate keypairs for program accounts (usually only need to be done once)
+```bash
+solana-keygen new -o <path>
+
+# examples (from root)
+solana-keygen new -o ./gauntlet/packages/gauntlet-solana-contracts/artifacts/programId/access_controller.json
+solana-keygen new -o ./gauntlet/packages/gauntlet-solana-contracts/artifacts/programId/deviation_flagging_validator.json
+solana-keygen new -o ./gauntlet/packages/gauntlet-solana-contracts/artifacts/programId/ocr2.json
+```
+
+Add the pubkeys to the their respective program (do for each program)
+```rust
+// example in programs/ocr2/src/lib.rs
+declare_id!("CF6b2XF6BZw65aznGzXwzF5A8iGhDBoeNYQiXyH4MWdQ");
+```
+
+Compile program artifacts (do each time contract changes)
+```bash
+# from root
+./tools/shell
+cd contracts
+anchor build
+
+# then exit environment
```

Comment:
```suggestion
# then copy build artifacts to Gauntlet
```

There should exist a helper script that does that for us - c/p all build artifacts to Gauntlet.

Reply:
https://github.com/smartcontractkit/chainlink-solana/issues/31